guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,385 @@
+# Updating from v12 to v13
+
+## Before you start
+
+v13 requires Node 14.x or higher to use, so make sure you're up to date. To check your Node version, use `node -v` in your terminal or command prompt, and if it's not high enough, update it! There are many resources online to help you with this step based on your host system.
+
+Once you've got Node up-to-date you can install v13 by running `npm install discord.js` in your terminal or command prompt for text-only use, or `npm install discord.js @discordjs/opus` for voice support.
+
+You can check your discord.js version with `npm list discord.js`. Should it still show v12.x uninstall (`npm uninstall discord.js`) and re-install discord.js and make sure the entry in your package.json does not prevent a major version update. Please refer to the [npm documentation](https://docs.npmjs.com/files/package.json#dependencies) for this.
```

Comment:
Happy to change the first line, but I don't think the second line should be removed. It's a good safety check if anything goes wrong installing (not that it really ever should)

Reply:
> there's no uninstallation needed

Technically true, practically somehow sometimes under certain conditions installing the new major "over" the old one, rather than removing it first has caused massive issues for some people in support on the v11 -> v12 switch. I personally could never reproduce any of it, but somehow people managed to end up with mixed versions where some files were still v11.